app-misc/vcontrold/vcontrold-0.98.5.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
```

Comment:
Did ARM arch team approve this? They are kinda sensitive about people adding `~arm` on their own.

Reply:
Well, I originally built this on my Raspberry Pi and have been using it without a problem for about two years, so I thought ~arm would be okay?

Reply:
(I think most people will use it on a Raspberry Pi or similar, as we can build an optical link device using the GPIO pins for a few € and a Raspberry Pi is just perfect for the job – so this ebuild really should have at least ~arm)

Reply:
@gentoo/arm